backend/users/models.py

```diff
@@ -1,15 +1,81 @@
 from django.contrib.auth.base_user import AbstractBaseUser
 from django.contrib.auth.models import PermissionsMixin
+from django.core import exceptions
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
+from model_utils import Choices
+from pycountry import countries
 
 from .managers import UserManager
 
+COUNTRIES = [{"code": country.alpha_2, "name": country.name} for country in countries]
+EU_COUNTRIES = (
```

Comment:
We have to decide if we want russia to be in the EU_COUNTRIES or not

Reply:
Maybe as EU countries we should consider only countries with the euro money??

Reply:
EU_COUNTRIES are those in CEE, so [this list](https://en.wikipedia.org/wiki/European_Union#Member_states). So they are right... :)